binary-search.py

```diff
@@ -309,7 +309,7 @@ def process_options ():
                         dest='traffic_generator',
                         help='name of traffic generator: trex-txrx or trex-txrx-profile or moongen-txrx of null-txrx',
                         default = "trex-txrx",
-                        choices = [ 'trex-txrx', 'trex-txrx-profile', 'moongen-txrx', 'null-txrx' ]
+                        choices = [ 'trex-txrx', 'trex-txrx-profile', 'moongen-txrx', 'xena', 'null-txrx' ]
```

Comment:
Adds xena as an argument option

Reply:
Should we add Xena to the help message as well?

Reply:
Sure.

Reply:
Updated.